src/utils/MultiInviter.js

```diff
@@ -111,17 +111,10 @@ export default class MultiInviter {
             }
 
             if (!ignoreProfile && SettingsStore.getValue("promptBeforeInviteUnknownUsers", this.roomId)) {
-                try {
-                    const profile = await MatrixClientPeg.get().getProfileInfo(addr);
-                    if (!profile) {
-                        // noinspection ExceptionCaughtLocallyJS
-                        throw new Error("User has no profile");
-                    }
-                } catch (e) {
-                    throw {
-                        errcode: "RIOT.USER_NOT_FOUND",
```

Comment:
we should be able to remove this error code from the other branches/checks now if we're not raising it. (before the diff this would be lines 174 and 215 - the if/else ladder in `_doInvite` and the `unknownProfileErrors` const respectively)